src/components/Footer/index.jsx

```diff
@@ -13,10 +13,10 @@ import TwitterIcon from "@mui/icons-material/Twitter";
 import MailIcon from "@mui/icons-material/Mail";
 
 const socialMediaLinks = [
-  { id: 0, path: "https://www.facebook.com", Icon: FacebookIcon },
-  { id: 1, path: "https://www.instagram.com", Icon: InstagramIcon },
-  { id: 2, path: "https://www.twitter.com", Icon: TwitterIcon },
-  { id: 3, path: "https://www.gmail.com", Icon: MailIcon },
+  { id: 0, path: "www.facebook.com", Icon: FacebookIcon },
+  { id: 1, path: "www.instagram.com", Icon: InstagramIcon },
+  { id: 2, path: "www.twitter.com", Icon: TwitterIcon },
+  { id: 3, path: "www.gmail.com", Icon: MailIcon },
```

Comment:
www is outdated pattern. Not all domains are handling www prefix. Don't use the www prefix.

Reply:
done